<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaleinosAdminMainWindow</class>
 <widget class="QMainWindow" name="BaleinosAdminMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BaleinosAdminMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-10</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string># Baleinos admin app üê≥</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sudo_label">
    <property name="geometry">
     <rect>
      <x>458</x>
      <y>8</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>mode sudo :</string>
    </property>
   </widget>
   <widget class="QLabel" name="sudo_indicator_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>8</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="message_content_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>671</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="message_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Message :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_policy_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>531</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>D√©sactiver les policies firefox (les utilisateurs peuvent modifier Firefox)</string>
    </property>
   </widget>
   <widget class="QLabel" name="exisintg_policies_file_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Le fichier existe :</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_indicator_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="homepage_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>216</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Page d'acceuil :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>351</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Sauvegarder / r√©activer les policies Firefox</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="homepage_edit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>411</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>https://impots.gouv.fr</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>254</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Moteur de recherche:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="engine_comboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>371</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="transfer_bookmarks_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Transf√©rer marque-pages</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Si vous avez configur√© les marques pages sur le compte admin et que vous voulez les transf√©rer au compte utilisateur :</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="exisintg_policies_file_label_2">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>52</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom du compte utilisateur :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_account_name_lineEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>personne</string>
    </property>
   </widget>
   <widget class="QLabel" name="exisintg_policies_file_path_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Policies folder :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="policies_path_lineEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>/etc/firefox/policies</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
